Makefile

```diff
@@ -128,6 +128,7 @@ build: generate fmt vet ## Build manager binary.
 
 .PHONY: run
 run: export ENABLE_WEBHOOKS?=false
+run: export OPERATOR_TEMPLATES=./templates/
```

Comment:
Let's change it to export OPERATOR_TEMPLATES?=./templates/ so it can be overridden from CLI

Reply:
Oh nice, TIL: https://www.gnu.org/software/make/manual/make.html#index-conditional-variable-assignment